src/ripple/app/consensus/LedgerConsensus.cpp

```diff
@@ -372,7 +374,8 @@ class LedgerConsensusImp
       @param map      the transaction set.
       @param acquired true if we have acquired the transaction set.
     */
-    void mapComplete (uint256 const& hash, SHAMap::ref map, bool acquired)
+    void mapComplete (uint256 const& hash, std::shared_ptr<SHAMap> const& map,
+                      bool acquired)
```

Comment:
My opinion is that it is an improvement because `ref` does not convey what the actual argument is.  `ref` is concise to the point of being obfuscated. The first time I read `ref` (a year ago), I thought it meant `SHAMap&`.